README.md

```diff
@@ -34,19 +35,29 @@ Consult the [Python SDK reference docs](https://kubeflow-pipelines.readthedocs.i
 
 ## Contributing to Kubeflow Pipelines
 
-Before you start contributing to Kubeflow Pipelines, read the guidelines in [How to Contribute](./CONTRIBUTING.md). To learn how to build and deploy Kubeflow Pipelines from source code, read the [developer guide](./developer_guide.md).
+Before you start contributing to Kubeflow Pipelines, read the guidelines in [How to Contribute](./CONTRIBUTING.md). To learn how to build and deploy Kubeflow Pipelines from source code, reagd the [developer guide](./developer_guide.md).
+
+## Kubeflow Pipelines Community
+
+### Community Meeting
 
+The Kubeflow Pipelines Community Meeting occurs every other Wed 10-11AM (PST).
 
-## Kubeflow Pipelines Community Meeting
+[Calendar Invite](https://calendar.google.com/event?action=TEMPLATE&tmeid=NTdoNG5uMDBtcnJlYmdlOWt1c2lkY25jdmlfMjAxOTExMTNUMTgwMDAwWiBqZXNzaWV6aHVAZ29vZ2xlLmNvbQ&tmsrc=jessiezhu%40google.com&scp=ALL)
 
-The meeting is happening every other Wed 10-11AM (PST)
-[Calendar Invite](https://calendar.google.com/event?action=TEMPLATE&tmeid=NTdoNG5uMDBtcnJlYmdlOWt1c2lkY25jdmlfMjAxOTExMTNUMTgwMDAwWiBqZXNzaWV6aHVAZ29vZ2xlLmNvbQ&tmsrc=jessiezhu%40google.com&scp=ALL) or [Join Meeting Directly](https://meet.google.com/phd-ixfj-kcr/)
+[Direct Meeting Link](https://zoom.us/j/92607298595?pwd%3DVlKLUbiguGkbT9oKbaoDmCxrhbRop7.1&sa=D&source=calendar&ust=1736264977415448&usg=AOvVaw1EIkjFsKy0d4yQPptIJS3x)
 
 [Meeting notes](http://bit.ly/kfp-meeting-notes)
 
-## Kubeflow Pipelines Slack Channel
+### Slack
 
-[#kubeflow-pipelines](https://kubeflow.slack.com)
+We also have a slack channel on the Cloud Native Computing Foundation Slack workspace at [#kubeflow-pipelines](https://cloud-native.slack.com/archives/C073N7BMLB1)
```

Comment:
Is it better to link to https://www.kubeflow.org/docs/about/community/#kubeflow-slack-channels instead?

https://cloud-native.slack.com/archives/C073N7BMLB1  or https://kubeflow.slack.com assumes someone is already a member of the slack workspace.